### PR TITLE
[MINOR] New Fields.CODE and Formatting

### PR DESCRIPTION
**Pull request by jaceklaskowski**
Minor auth-related changes:

1. New Fields.CODE
2. Use Fields constants whenever possible
3. Formatting

### CODE REVIEW

**Comment by jaceklaskowski** [comment]
@creechy Please review 🙏

File: server/src/main/java/io/unitycatalog/server/service/CatalogService.java [hunk 1]
```diff
@@ -53,7 +53,7 @@ public HttpResponse createCatalog(CreateCatalog createCatalog) {
   }
 
   @Get("")
-  @AuthorizeExpression("#defer")
+  @AuthorizeExpression
```
Comment (creechy) [inline server/src/main/java/io/unitycatalog/server/service/CatalogService.java:56]: not sure what this change is about. It doesn't really matter either way but using `#defer` helps identify that the authorization is intentionally done elsewhere/later.

**Review by creechy — CHANGES_REQUESTED** [review]
Just one question, otherwise looks good.

File: server/src/main/java/io/unitycatalog/server/service/CatalogService.java (hunk 1 above)
Comment (jaceklaskowski) [reply server/src/main/java/io/unitycatalog/server/service/CatalogService.java:56]: Okey, I'll revert this change. I think it was IDEA to propose it as `#defer` is the default value.

**Review by creechy — APPROVED** [review]

**Pull request merged**